jobs/config.json

```diff
@@ -6767,6 +6770,8 @@
       "--gcp-node-image=gci",
       "--gcp-project=kubernetes-scale",
       "--gcp-zone=us-east1-a",
+      "--gke-command-group=beta",
+      "--gke-create-command=container clusters create --quiet --enable-ip-alias --create-subnetwork=ip-alias-subnet --cluster-ipv4-cidr=/12 --services-ipv4-cidr=/19",
```

Comment:
same as above, the /12 would allow only a max of 4K nodes. Is that the intent?

Reply:
Same as above.